src/main/java/by/bookify/book/repository/BookRepository.java

```diff
@@ -0,0 +1,11 @@
+package by.bookify.book.repository;
+
+import by.bookify.book.model.Book;
+import org.springframework.data.repository.CrudRepository;
+
+import java.util.List;
+
+public interface BookRepository extends CrudRepository<Book, Long> {
```

Comment:
Why CrudRepository and not a jpa one. With crud you are not be able to paging

Reply:
Done